src/coreclr/pal/src/arch/amd64/asmconstants.h

```diff
@@ -91,7 +93,8 @@
 #define CONTEXT_KMask0 CONTEXT_Ymm0H+(16*16)
 #define CONTEXT_Zmm0H CONTEXT_KMask0+(8*8)
 #define CONTEXT_Zmm16 CONTEXT_Zmm0H+(32*16)
-#define CONTEXT_Size CONTEXT_Zmm16+(64*16)
+#define CONTEXT_Egpr  CONTEXT_Zmm16+(16*8)
+#define CONTEXT_Size CONTEXT_Egpr+(64*16)
```

Comment:
This doesn't look right. It seems it should be
```
#define CONTEXT_Egpr CONTEXT_Zmm16+(64*16)
#define CONTEXT_Size CONTEXT_Egpr+(16*8)
```

Reply:
Looks like this comment is still pending.

I'd agree with the janvorli's assessment here. Each new define is `previousDefine+(SizeOfPreviousDefine)` and `_Zmm16` should have a size of `64*16`, while `_Egpr` should have a size of `8*16` (typically `SizeOfPreviousDefine` is built up as `SizePerItem*NumberOfItems`, so `64*16` because each register is 64-bytes and there are 16 of them, similarly `8*16` because each register is 8-bytes and there are 16 of them).

Reply:
Thanks for pointing out and the explanation!

sorry for the delay, we had a holiday yesterday.